src/Engine/ProtoCore/Lang/BuiltInFunctionEndPoint.cs

```diff
@@ -1200,7 +1220,60 @@ internal static StackValue[] GenerateRangeByAmount(decimal start, long amount, d
                         }
                 }
             }
-            return range == null ? StackValue.Null : runtimeCore.RuntimeMemory.Heap.AllocateArray(range);
+
+            return null;
+        }
+
+
+        private static StackValue[] GenerateAlphabetRange(StackValue svStart, StackValue svEnd, StackValue svStep, RuntimeCore runtimeCore)
+        {
+            if (!svStart.IsString || !svEnd.IsString || !svStep.IsInteger)
+            {
+                runtimeCore.RuntimeStatus.LogWarning(
+                    WarningID.kInvalidArguments,
+                    Resources.kInvalidArgumentsInRangeExpression);
+                return null;
+            }
+
+            var startValue = runtimeCore.Heap.ToHeapObject<DSString>(svStart).Value;
+            var endValue = runtimeCore.Heap.ToHeapObject<DSString>(svEnd).Value;
+
+            // Start and end values can be just alphabet letters. So their length can't be more than 1.
+            if (startValue.Length != 1 || endValue.Length != 1)
+            {
+                runtimeCore.RuntimeStatus.LogWarning(
+                    WarningID.kInvalidArguments,
+                    Resources.kInvalidArgumentsInRangeExpression);
+                return null;
+            }
+
+            var startLetter = startValue.ToCharArray().First();
+            var endLetter = endValue.ToCharArray().First();
+            int step = Convert.ToInt32(svStep.RawIntValue);
+
+            // Alphabet sequence can be made just from letters.
+            if (!Char.IsLetter(startLetter) || !Char.IsLetter(endLetter) || step <= 0)
+            {
+                runtimeCore.RuntimeStatus.LogWarning(
+                    WarningID.kInvalidArguments,
+                    Resources.kInvalidArgumentsInRangeExpression);
+                return null;
+            }
+
+            StackValue[] letters;
+            int stepOffset = (startLetter < endLetter) ? 1 : -1;
+            int stepnum = (int)Math.Abs(Math.Truncate((endLetter - startLetter) / (double)step)) + 1;
+
+            letters = Enumerable.Range(1, stepnum)
+                // Generate arithmetic progression.
+                 .Select(x => startLetter + (x - 1) * step * stepOffset)
+                // Take just letters.
+                 .Where(x => Char.IsLetter((char)x))
+                // Create stack values.
+                 .Select(x => StackValue.BuildChar((char)x))
```

Comment:
Here we should return string instead of char because 1) char type will be removed 2) normally the result will be used in other string operation like concatenation.